client/src/components/course-view/NewAttainmentDialog.tsx

```diff
@@ -36,10 +37,13 @@ const AddAttainmentDialog = ({
       <DialogTitle>Create new attainment</DialogTitle>
       <DialogContent>
         <TextField
-          sx={{mt: 1}}
+          sx={{mt: 1, '&input:user-invalid': {borderColor: 'red'}}}
```

Comment:
Is this necessary? It seems to work without it

Reply:
fixed